src/testing/AddTestToIdentifierPage.tsx

```diff
@@ -85,7 +85,9 @@ export const AddTestToIdentifierPage: FC = () => {
   });
 
   const fieldError = (key: keyof FormFields) =>
-    form.touched[key] && form.errors[key] ? <Text>{form.errors[key]}</Text> : null;
+    (form.submitCount > 0 || form.touched[key]) && form.errors[key] ? (
```

Comment:
Seen this logic three times now, maybe you should move to a library?

Reply:
Yup, will see if I can create a type-safe component 👍